synapse/cores/common.py

```diff
@@ -1938,3 +1912,68 @@ def _tufosByInetCidr(self, prop, valu, limit=None):
 
         return self.getTufosBy('range', prop, (ipv4addr, ipv4addr+mask), limit=limit)
 
+    #############################################################
```

Comment:
This was done to facilitate remote access to the data model API on a cortex. Without it, we'd basically have to create and maintain wrapper methods on the cortex for every data model API. Lesser of two evils. :)